update M.A.App (#14519)
aspnet/AspNetCore.Docs: aspnetcore/fundamentals/metapackage.md

---
# Microsoft.AspNetCore.All metapackage for ASP.NET Core 2.0

::: moniker range=">= aspnetcore-3.0"

> The `Microsoft.AspNetCore.All` metapackage is not included in ASP.NET Core 3.0 and later. See [this GitHub issue](https://github.com/aspnet/Announcements/issues/314) for more information.

Reviewer: ```suggestion
The `Microsoft.AspNetCore.All` metapackage isn't included in ASP.NET Core 3.0 and later. For more information, see [this GitHub issue](https://github.com/aspnet/Announcements/issues/314).
```

> ::: moniker-end

Reviewer: ```suggestion

::: moniker-end
```
